src/hotspot/share/utilities/rbTree.hpp

```diff
@@ -116,21 +144,54 @@ class RBTree {
 
   typedef TreeType::RBNode NodeType;
 
+  // Represents the location of a (would be) node in the tree.
+  // If a cursor is valid (valid() == true) it points somewhere in the tree.
+  // If the cursor points to an existing node (found() == true), node() can be used to access that node,
+  // Otherwise nullptr is returned, regardless if the node is valid or not.
```

Comment:
Style: "nullptr" should be "null" here.

Reply:
"Otherwise", but the above line has a comma.  What can be used to access that node? You don't say :-).